# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/blackbox/hotfix-time.h.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

diff --git a/src/Toolbar.cc b/src/Toolbar.cc
index e88b8f6b..8ccad201 100644
--- a/src/Toolbar.cc
+++ b/src/Toolbar.cc
@@ -38,6 +38,7 @@
 #include <X11/Xutil.h>
 #include <sys/time.h>
 #include <assert.h>
+#include <time.h>
 
 
 long nextTimeout(int resolution)
